features/102.ngsild.working_with_context.feature

```diff
@@ -0,0 +1,82 @@
+Feature: test tutorial 102 NGSI-LD: Working with Context feature
+
+  This is the feature file of the FIWARE Step by Step tutorial for NGSI-LD
+  url: https://ngsi-ld-tutorials.readthedocs.io/en/latest/working-with-%40context.html
+  git-clone: https://github.com/FIWARE/tutorials.Getting-Started.git
+  git-directory: /tmp/tutorials.Getting-Started
+  shell-commands: git checkout NGSI-LD ; ./services orion
+  clean-shell-commands: ./services stop
+
+  Background:
+    Given I set the tutorial 102 NGSI-LD
+
+# Once the services have started up, and before interacting with the context broker itself, it is useful to check
+# that the necessary prerequisites are in place.
+  Scenario Outline: Check the files are in place.
+    When I prepare a GET HTTP request to "http://localhost:3004/<file>"
+    And   I perform the query request
+    Then  I receive a HTTP "200" response code
+    Examples:
+       | file                     |
+       | ngsi-context.jsonld      |
+       | json-context.jsonld      |
+       | alternate-context.jsonld |
+
+  # Request 1
+  Scenario: Checking the Service Health
+    When I prepare a GET HTTP request to "http://localhost:1026/version"
+    And   I perform the query request
+    Then  I receive a HTTP "200" response code
+
+  # Request 2: Create data entities
+  Scenario: Create a new data entity
+    When  I prepare a POST HTTP request to "http://localhost:1026/ngsi-ld/v1/entities/"
+    And   I set header Content-type to application/ld+json
+    And   I set the body request as described in 102.ld.02.request.json
```

Comment:
Please, use " for the file name to follow the style

Reply:
A technical debt (Issue #34) was created to tackle with this issue in the next releases.

NTC